apps/mobile/src/app/components/Auction/AuctionTemplate.tsx

```diff
@@ -13,21 +17,48 @@ export function AuctionTemplate(props: Props) {
   const { auction, isLoading } = props;
   const insets = useSafeAreaInsets();
 
+  //TODO
+  const descTmp =
+    'Auction offer details Lorem ipsum dolor sit amet. Aut atque voluptas in corporis accusamus sit iure officiis aut saepe quos et maiores nemo. Non unde enim sit quia quas est dolorum consequatur vel temporibus nostrum. Vel eveniet unde et corporis obcaecati et harum soluta non quia aliquid et autem quas ea itaque galisum. Ex accusamus voluptatibus non unde beatae quo dolor dolores non perspiciatis neque.';
+
   if (!auction) {
     return <Text>Loading...</Text>;
   }
 
   return (
-    <View mt={10} px={8} paddingTop={insets.top} paddingBottom={insets.bottom}>
-      <View mb={5}>
-        <Link to='/'>
-          <Text>Auctions</Text>
-        </Link>
+    <>
+      <AuctionHeader />
+      <ImageBackground
```

Comment:
Maybe styling for placeholder could be in the component itself? if there is no img then show grey for now etc. Also height could be applied with styled components as separate component like <SingleAuctionImage>